free-2.md

```diff
@@ -35,10 +61,27 @@ user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
 
+Once events and TODOs are in, getting the information out should also be easy.
+Ideally, there will be some levels of reminders that go from "this is what you
+should take care of today" to "you have to start this assignment now or else
+you'll skip the talk you really want to go to later."
+
+I feel this is the right way because it's how my brain works. When I see a new
+thing I have to do, I only think of it for so long before it leaves my mind so
+I want to get it down as soon as possible. I also am often asking "Do I have
+enough free time to play a few games?" or "I'm feeling productive, what do I
+need to do now?" This system could solve both.
+
```

Comment:
I agree. This would be great. It would also be nice to be able to associate resources with tasks. For example, "here's a link to the PDF of the reading."